### PR TITLE
[HOTFIX - MERGE WITH GITHUB] 5977 Map Tiles Redirects

### LINKED ISSUE #5977: Switch domains to address redirect issue and remove text from election map
**What we're after:**
Election search map is displaying text because we need to switch domains to Stadia. We previously fixed the July 31 issue when Stamen started redirecting to Stadia. Now Stamen is going to stop redirecting so we'll need to switch to using Stadia directly.

Resources: 
- https://docs.stadiamaps.com/guides/migrating-from-stamen-map-tiles/#frameworklibrary-migration-steps
- https://github.com/leaflet-extras/leaflet-providers/blob/master/README.md

### Completion criteria
- [ ] Switched to Stadia and text no longer displays on the election search map

### PR DESCRIPTION
**Pull request by rfultz**
## Summary

- Resolves #5977

Trying to solve the issue of Stamen maps being moved to Stadia. Everything is in place on Feature and seems to be working as expected—functionality and Stadia's dashboard is showing usage.

### Required reviewers

Two? Feel free to add/remove

## Impacted areas of the application

**Sitewide**
This sets `SECURE_REFERRER_POLICY` to `"strict-origin-when-cross-origin"` so external sites can see our domain name when we make requests from them (instead of `null`), necessary for validating with stadiamaps.com

**Maps**
Election search maps at
- [/](http://127.0.0.1:8000/)
- [/data/](http://127.0.0.1:8000/data/)
- [/data/elections/](http://127.0.0.1:8000/data/elections/)

## Screenshots

BROKEN
image

image

FIXED
image

image

## Related PRs

Not really

## How to test

1) Check
   a. Feature, or
   b. localhost (pull the branch locally, `npm i`, and `npm run build`)
2) play with the map, dragging and zooming it
3) let me know if you see
   - any of the error messages and
   - if a reload fixes it and
   - if another shift-reload breaks it again

### CODE REVIEW

File: fec/home/templates/purgecss-homepage/full.html [hunk 1]
```diff
@@ -401,12 +401,12 @@ <h2 class="heading--section u-padding--top">Compare candidates in an election</h
...
-        <img alt="Map tile image" role="presentation" src="https://stamen-tiles-d.a.ssl.fastly.net/toner-lite/3/1/2.png" class="leaflet-tile leaflet-tile-loaded">
-        <img alt="Map tile image" role="presentation" src="https://stamen-tiles-a.a.ssl.fastly.net/toner-lite/3/1/3.png" class="leaflet-tile leaflet-tile-loaded">
-        <img alt="Map tile image" role="presentation" src="https://stamen-tiles-c.a.ssl.fastly.net/toner-lite/3/0/2.png" class="leaflet-tile leaflet-tile-loaded">
-        <img alt="Map tile image" role="presentation" src="https://stamen-tiles-a.a.ssl.fastly.net/toner-lite/3/2/2.png" class="leaflet-tile leaflet-tile-loaded">
-        <img alt="Map tile image" role="presentation" src="https://stamen-tiles-d.a.ssl.fastly.net/toner-lite/3/0/3.png" class="leaflet-tile leaflet-tile-loaded">
-        <img alt="Map tile image" role="presentation" src="https://stamen-tiles-b.a.ssl.fastly.net/toner-lite/3/2/3.png" class="leaflet-tile leaflet-tile-loaded">
+        <img alt="Map tile image" role="presentation" src="/stamen-tiles-d.a.ssl.fastly.net/toner-lite/3/1/2.png" class="leaflet-tile leaflet-tile-loaded">
```
Comment (rfultz) [inline fec/home/templates/purgecss-homepage/full.html:404]: The purgecss files are only used during build to optimize homepage css

File: package.json [hunk 2]
```diff
@@ -105,7 +104,8 @@
     "karma-coverage-istanbul-reporter": "^2.0.5",
     "karma-mocha": "^0.2.2",
     "karma-webpack": "^3.0.5",
-    "leaflet": "^1.7.1",
+    "leaflet": "^1.9.4",
+    "leaflet-providers": "^2.0.0",
```
Comment (rfultz) [inline package.json:108]: Had to update leaflet-providers so updated leaflet, too

**Review by JonellaCulmer — APPROVED** [review]
No messages appeared during my review of it on feature, include on zoom, drag, or click down to district level view. Thank you, @rfultz

**Review by tmpayton — APPROVED** [review]
Great work @rfultz! Tested locally and everything looks good.

**Pull request merged**